errormapper/errormapper.go

```diff
@@ -0,0 +1,68 @@
+package errormapper
```

Comment:
Shouldn't this be in an other repo if it is used from different steps? I think we shouldn't introduce dependency between steps just to use "utility" kind of stuff